src/overlays/actors/ovl_Door_Ana/z_door_ana.c

```diff
@@ -63,11 +63,11 @@ void DoorAna_SetupAction(DoorAna* this, DoorAnaActionFunc actionFunction) {
 
 void DoorAna_Init(Actor* thisx, GlobalContext* globalCtx) {
     DoorAna* this = THIS;
-    s32 grottoType = GET_DOORANA_TYPE(this);
+    s32 grottoType = DOORANA_GET_TYPE(this);
```

Comment:
```suggestion
    s32 grottoType = DOORANA_GET_TYPE(this);
    
```

Reply:
This seems a little unclear, as I didn't grab the next line, but this should just add a newline to seperate variable declarations from code. I.e, it would become 
```c
    DoorAna* this = THIS;
    s32 grottoType = DOORANA_GET_TYPE(this);

    this->actor.shape.rot.y = this->actor.shape.rot.z = 0;
```